### DISCUSSION TITLE
I want to learn about cache to speed up my builds

### DISCUSSION THREAD

**Discussion by dddssw** [webpack/webpack#18250] [Q&A] [answered]
I'm new to webpack and I want to know when the cache will expire after caching is enabled. I know it has a fixed expiration time, and some dependency files can be configured to control whether it expires. But if I modify my business code, will the cache automatically expire? If so, will the entire cache expire or part of the cache expire? I'm worried that the product is old due to caching，Thank you.

**Comment by alexander-akait** [comment, marked as answer]
No, webpack invalidate cache only for changed files/modules (and files/modules where this module was used)

**Reply by dddssw** [reply]
Just asking,Will synchronous import and asynchronous import of a module cause the module to be packaged repeatedly? Setting chunks in splitChunks to all should prevent it from being packaged repeatedly? What will happen if it is set to initial? Will it only affect the entry file?

**Reply by alexander-akait** [reply]
Your chunk graph need to rebuild, so cache will be used partially
